src/components/ReleaseNotesVersions/ReleaseNotesVersions.module.scss

```diff
@@ -0,0 +1,12 @@
+.dropdownContainer {
+  display: flex;
+  justify-content: center;
+}
+
+.dropdownContainer > div:first-child {
```

Comment:
Did you want to keep this or style with `container` in `customSelectStyles`?

Reply:
Yeah, just to avoid having to add more JS to figure out the screensize.